courses/free-courses-th.md

```diff
@@ -47,6 +47,7 @@
 
 * [จาวาสคริปต์เบื้องต้น](https://phyblas.hinaboshi.com/saraban/javascript) - Phyblas
 * [ภาษา JavaScript](http://marcuscode.com/lang/javascript) - MarcusCode
+* [สอนพื้นฐาน JavaScript ทั้งหมดแบบจบในคลิปเดียว !! 🔥](https://youtu.be/PGZ7QiKdumo) - BorntoDev
```

Comment:
- Not a playlist but +3h video with chapters in the timeline
- Stripping emojis
- Expanding YouTube urls

```suggestion
* [สอนพื้นฐาน JavaScript ทั้งหมดแบบจบในคลิปเดียว !!](https://www.youtube.com/watch?v=PGZ7QiKdumo) - BorntoDev
```